src/sentry/static/sentry/app/components/avatar/suggestedAvatarStack.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {css} from '@emotion/core';
+import styled from '@emotion/styled';
+
+import ActorAvatar from 'app/components/avatar/actorAvatar';
+import BaseAvatar from 'app/components/avatar/baseAvatar';
+import {Actor} from 'app/types';
+
+type Props = {
+  owners: Actor[];
+};
+
+// Constrain the number of visible suggestions
+const MAX_SUGGESTIONS = 5;
+
+const SuggestedAvatarStack = ({owners, ...props}: Props) => {
+  const backgroundAvatarProps = {
+    round: owners[0].type === 'user',
+    suggested: true,
+    ...props,
+  };
+  const reversedOwners = owners.slice(0, MAX_SUGGESTIONS).reverse();
+  return (
+    <AvatarStack>
+      {reversedOwners.map((owner, id) => {
+        if (id === reversedOwners.length - 1) {
```

Comment:
Why only show one owner?

Reply:
hmm this is just from the mocks @robinrendle created, but I imagine showing multiple under one another would be kind of hard to see. And the goal is just to show the user there are suggested options which they can pick from in the dropdown:

![image](https://user-images.githubusercontent.com/9372512/100660522-27861200-3320-11eb-8b0d-633dbb17c521.png)

We can discuss this though if you think showing more would be useful

Reply:
Ah, in that case, I would change this logic so that we only render the last el of `owners` to avoid the array copy + reverse and then generate the background avatars separately.

Reply:
oooh good stuff yea that is a lot nicer than what I have going on